matlab_kernel/kernel.py

```diff
@@ -267,7 +269,7 @@ def _execute_sync(self, code):
         except (SyntaxError, MatlabExecutionError) as exc:
             stdout = exc.args[0]
             self.Error(stdout)
-            return ExceptionWrapper("Error", -1, stdout)
```

Comment:
I've not tested the effect of this, but it hopes to fix the `-1` JSON errors in Calysto#121. It looks like MATLAB is always executed as async. How could I test this?

Reply:
Probably it's best if I don't touch this. It still works in that case. See commit 865dfa65edbe8790ebde0eab8332a252b6032065